source/main-hw.h

```diff
@@ -47,6 +47,32 @@ extern Serial shared_pc;
         {SPI0,   sizeof(*SPI0),   UVISOR_TACLDEF_PERIPH}, \
     }
 
+#elif defined(TARGET_DISCO_F429ZI)
+
+#define LED_ON  true
+#define LED_OFF false
+
+#define MAIN_ACL(acl_list_name) \
+    static const UvisorBoxAclItem acl_list_name[] = { \
+        {GPIOA,               sizeof(*GPIOA),  UVISOR_TACLDEF_PERIPH}, \
+        {GPIOB,               sizeof(*GPIOB),  UVISOR_TACLDEF_PERIPH}, \
+        {GPIOC,               sizeof(*GPIOC),  UVISOR_TACLDEF_PERIPH}, \
+        {GPIOD,               sizeof(*GPIOD),  UVISOR_TACLDEF_PERIPH}, \
+        {GPIOE,               sizeof(*GPIOE),  UVISOR_TACLDEF_PERIPH}, \
+        {RTC,                 sizeof(*RTC),    UVISOR_TACLDEF_PERIPH}, \
+        {TIM5,                sizeof(*TIM5),   UVISOR_TACLDEF_PERIPH}, \
+        {USART1,              sizeof(*USART1), UVISOR_TACLDEF_PERIPH}, \
+        {I2C1,                sizeof(*I2C1),   UVISOR_TACLDEF_PERIPH}, \
+        {SPI1,                sizeof(*SPI1),   UVISOR_TACLDEF_PERIPH}, \
+        {RCC,                 sizeof(*RCC),    UVISOR_TACLDEF_PERIPH}, \
+        {FLASH,               sizeof(*FLASH),  UVISOR_TACLDEF_PERIPH}, \
+        {PWR,                 sizeof(*PWR),    UVISOR_TACLDEF_PERIPH}, \
+        {EXTI,                sizeof(*EXTI),   UVISOR_TACLDEF_PERIPH}, \
+        {GPIOG,               sizeof(*GPIOG),  UVISOR_TACLDEF_PERIPH}, \
+        {SYSCFG,              sizeof(*SYSCFG), UVISOR_TACLDEF_PERIPH}, \
+        {(void *) 0x42000000, 0x01000000,      UVISOR_TACLDEF_PERIPH}, /* FIXME */ \
```

Comment:
Add a bit more detail to your FIXME comment. It is not clear what you are thinking needs fixing.